env.example

```diff
@@ -213,4 +213,15 @@ JIBRI_XMPP_PASSWORD=
 #RESTART_POLICY=unless-stopped
 
 # Jitsi image version (useful for local development)
-#JITSI_IMAGE_VERSION=latest
+JITSI_IMAGE_VERSION=stable-8252
```

Comment:
Please cleanup the PR and just leave the minimum necessary. I understand you made these changes for your own deployment, but that is not something we can commit to master.